kraken-app/kraken-app-agent/src/main/resources/application.yaml

```diff
@@ -92,13 +92,19 @@ app:
   controlPlane:
     url: http://localhost:8001
     token: Bearer ${KRAKEN_MGMT_API_ACCESS_TOKEN:FAKE_TOKEN}
+  external-system:
+    enabled: false
+    url: ${EXTERNAL_SYSTEM_PUSH_LOG_URL:http://localhost:8888}
```

Comment:
woudl u like to remove external.url and token and reuse the control plane.url and token.
```yaml
controlPlane:
  push-activity-log-external:
        enabled: false

  

```

Reply:
Hi @kuangxiang20240501 
The changes have been applied. Please verify them.